### PR TITLE
EC-30: Refactor CLI argument's processing

### PR DESCRIPTION
**Pull request by a-ruban**

### CODE REVIEW

File: ess_base/providers/cli/utils.py [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+        plain_method_data (str): data from user input.
 
-def _execute_handler_method(cli_handler, method_name, method_arguments):
+    Returns:
+        args (list): positional arguments.
+        kwargs (dict): keyword arguments.
+    """
+    method_data = plain_method_data.replace(' ', '').split(',')
+    args = []
+    kwargs = {}
```
Comment (dmytrostriletskyi) [inline ess_base/providers/cli/utils.py]: Make it one line. Also use blank line to separate logic blocks, please.

File: ess_base/providers/cli/utils.py [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
-    if isinstance(command_result, dict):
-        command_result = json.dumps(command_result, indent=4, sort_keys=True)
+    if args and kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(*args, **kwargs)
+    elif args and not kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(*args)
+    elif not args and kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(**kwargs)
+    else:
+        raise AttributeError
```
Comment (dmytrostriletskyi) [inline ess_base/providers/cli/utils.py]: Raise error with some message, please.

File: ess_base/providers/cli/utils.py [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
-    command_result = getattr(cli_handler(), method_name_formatted)(**method_arguments)
 
-    if isinstance(command_result, dict):
-        command_result = json.dumps(command_result, indent=4, sort_keys=True)
+    if args and kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(*args, **kwargs)
+    elif args and not kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(*args)
+    elif not args and kwargs:
+        command_result = getattr(cli_handler(), method_name_formatted)(**kwargs)
```
Comment (dmytrostriletskyi) [inline ess_base/providers/cli/utils.py]: Are we able to follow DRY principle here for `getattr(cli_handler(), method_name_formatted)`? For example, make individual variable for it.

File: ess_base/providers/cli/utils.py (hunk 2 above)
Comment (a-ruban) [reply ess_base/providers/cli/utils.py]: @dmytrostriletskyi done.

File: ess_base/providers/cli/utils.py (hunk 1 above)
Comment (a-ruban) [reply ess_base/providers/cli/utils.py]: @dmytrostriletskyi done.

File: ess_base/providers/cli/utils.py (hunk 3 above)
Comment (a-ruban) [reply ess_base/providers/cli/utils.py]: @dmytrostriletskyi done.

**Pull request merged**